Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -536,6 +536,8 @@ extension BrowserViewController: WKNavigationDelegate {
         return (.cancelAuthenticationChallenge, nil)
       }
     }
+    
+    // TODO: Basic Auth Check
```

Comment:
what about this todo?

Reply:
Ah it is actually handled right there but forgot to remove that comment later. Fix at https://github.com/brave/brave-ios/pull/7163/commits/cfbe09be4fe07a51c3b867ad92d39c66224c918a